common-features/adaptive-rendering.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Adaptive Rendering
+page_title: Adaptive Rendering
+description: Explore how the components with popup elements can react to the changes in the
+slug: adaptive-rendering
+tags: telerik,blazor,adaptive,rendering,mobile
+published: True
+position: 0
+---
+
+# Adaptive Rendering
+
+Telerik UI for Blazor supports adaptive rendering for the components that incorporate popup elements. This functionality allows the component adapt to the screen size by providing different rendering of the popup element based on the screen dimensions.
+
+>caption In this article:
+
+* [Supported components](#supported-components)
+* [Basics](#basics)
+* [Rendering specifics](#rendering-specifics)
+
+## Supported components
+
+The adaptive rendering functionality is supported by the following components that incorporate popup elemens:
+
+* [AutoComplete]({%slug autocomplete-overview%})
+* [ComboBox]({%slug components/combobox/overview%})
+* [DropDownList]({%slug components/dropdownlist/overview%})
+* [MultiColumnComboBox]({%slug multicolumncombobox-overview%})
+* [MultiSelect]({%slug multiselect-overview%})
+
+## Basics
+
+To enable the adaptive rendering of the component use the `AdaptiveMode` parameter. It takes a member of the `AdaptiveMode` enum:
+
+* `None` (default)
+* `Auto`
+
+>caption Enable the adaptive rendering
+
+````CSHTML
+// NOTE: The configurations below includeс only the DropDownList, but is applicable to all of teh above listed components
+
+// Adapts to the screen size to use the appropriate rendering.
+<TelerikDropDownList ... AdaptiveMode="@AdaptiveMode.Auto" ... />
+````
+
+# Rendering specifics
+
+When you set the `AdaptiveMode` to `Auto`, the component will take the screen size into consideration to use the appropriate rendering. The different rendering essentially targets the popup element of the component and how it will be displayed to the user.
```

Comment:
The different rendering <del>essentially</del> targets the popup element of the component and how it will be displayed to the user.